### PR TITLE
update AboutUs

### PR DESCRIPTION
**Pull request by niteshkmdev**
Updated AboutUs. added Profile Photo's from LinkedIn. added Social Media Profile Link's and minor changes don't affect the working of the website.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -8,6 +8,7 @@
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
   <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+  <script src="https://kit.fontawesome.com/101c647319.js" crossorigin="anonymous"></script>
```
Comment (akshitarora921) [inline public/index.html]: we are already using react-icons. please use that.

Comment (niteshkmdev) [reply public/index.html]: > we are already using react-icons. please use that.

(mistakenly clicked on resolve conversation) anyway added react-icon have a look @akshitarora921

**Comment by akshitarora921** [comment]
Looks great! Thanks for the PR.

**Comment by akshitarora921** [comment]
Merging

**Pull request merged**